src/core.js

```diff
@@ -1,6 +1,44 @@
 'use strict';
+import {Model} from './model';
+import {serialize, deserialize} from './serializers/xml';
+import {persist} from './adapters/mediawiki';
+
+export {Core as InfoboxTemplateBuilder};
```

Comment:
I haven't tested it, I just read https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Hoisting so I thought I'd ask.